src/include/sof/trace/trace.h

```diff
@@ -245,10 +245,12 @@ do {								\
 	(void)id_1;						\
 	(void)id_2;						\
 	struct timeval tv;					\
+	char slevel[5][4] = {"dis", "err", "wrn", "inf", "dbg"}; \
 	char *msg = "(%s:%d) " format;				\
-	if (level >= host_trace_level) {			\
+	if (level <= host_trace_level) {			\
```

Comment:
I forgot to mention in commit text that this test was fixed. Higher priority traces have lower integer values.